insights/parsers/xfs_info.py

```diff
@@ -104,54 +130,56 @@ def parse_content(self, content):
                              )
         keyval_re = re.compile(r'(?P<key>[\w-]+)=(?P<value>\d+(?: blks)?)')
 
-        xfs_info = {}
         sect_info = None
         self.mount = ''
 
+        info_re = re.compile(r'^(?P<section>[\w-]+)?\s*' +
+                             r'=(?:(?P<specifier>\S+)(?:\s(?P<specval>\w+))?)?' +
+                             r'\s+(?P<keyvaldata>\w.*\w)$')
+        keyval_re = re.compile(r'(?P<key>[\w-]+)=(?P<value>\d+(?: blks)?)')
+
```

Comment:
Line 136-139 are duplicated with line 127-131

Reply:
Problems resolving merge conflicts - thanks for spotting that @xiangce